app/features/routes.py

```diff
@@ -0,0 +1,75 @@
+from app import db
+from app.features import bp
+from flask import render_template, request
+from flask_login import login_required
+from app.dummy_feature.models import Sensor, sensorFeature, SensorType, TemperatureData, Feature, Room
+import random
+import json
+import datetime
+
+
+# Feature selection page
+@bp.route('/feature_select')
+@login_required
+def feature_selector():
+    features = Feature.query.all()
+    if len(features) <= 0:
+        sensor = Sensor.query.get(1)
+        feature = Feature(name='Temperature Pot de fleur', enum_type='temperature')
+        feature.sensors.append(sensor)
+        db.session.add(feature)
+        db.session.commit()
+        features = Feature.query.all()
```

Comment:
Variable "features" is rendered to the template, but it's not checked, if features is null, HTML code will be crash.